paddle/fluid/framework/ir/graph_helper.cc

```diff
@@ -32,7 +32,7 @@ namespace framework {
 namespace ir {
 namespace {
 void SortHelper(
-    const std::map<ir::Node *, std::unordered_set<ir::Node *>> &adj_list,
+    const std::map<ir::Node *, std::vector<ir::Node *>, ir::NodeComp> &adj_list,
```

Comment:
try just change unordered_set to `std::set` ? 

the `set` is a binary search tree and should work like a sorted vector in this scenario.

Reply:
Done, it should be OK. please wait CI's result.

Reply:
Just now all CI testing are passed. Please help us review it again.